CMSgov/mint-app
# [NOREF] Remove Tables scrollability

# NOREF

## Changes and Description

- Updated `table td` styles to have wrap the white-space 
- Update snaps




## How to test this change

1. Check the app to ensure that tables are not expanding and scrolling pass the width
2. Add a solution to the IT Solution and ensure that the words are wrapping

|     |      |
|---|---|
|Before|<img width="1205" alt="image" src="https://github.com/CMSgov/mint-app/assets/2349702/85f9baed-97d6-4b0e-8f54-d054e4e6cd43">|
|After|<img width="1191" alt="image" src="https://github.com/CMSgov/mint-app/assets/2349702/b7fdf9fa-5b03-461e-8672-c9c37ed9b584">|





## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
